proxies-es6/demo.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+// Proxying a normal object
+var target = {};
+var handler = {
+  get(target, property, receiver) {
```

Comment:
ESLint fails with

```
/home/travis/build/GoogleChrome/samples/proxies-es6/demo.js
  6:25  error  "receiver" is defined but never used  no-unused-vars
```

I see the benefit of including `receiver` as a parameter so that developers know it'll get passed in, so if you're not going to make use of it in the method body, could you add in a `// eslint-disable-line no-unused-vars` at the end of the line?